spectacles/runner.py

```diff
@@ -126,7 +137,10 @@ async def __aenter__(self) -> None:
             self.branch = prod_state.branch
             self.commit = prod_state.commit
             if self.ephemeral:
-                self.branch = await self.checkout_temp_branch(prod_state.commit)
+                if self.use_personal_branch:
```

Comment:
Instead of having to use this if/else every time, it would be tidier to have one function that checks out a "temp" branch, with a parameter that controls whether or not it's the personal branch.